### DISCUSSION TITLE
ArgumentType.Word Keeps sending me null

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#1951] [Help]
I want to retrieve player names, and player names from players from another server.
That's why I wrote this code:
```
val inputPlayer = ArgumentType.Word("inputPlayer")
        inputGroup.setSuggestionCallback { _, _, suggestion ->
            ServerManager.getPlayers().forEach {
                suggestion.addEntry(SuggestionEntry(it.value))
            }
        }
``` But when I run this command in game, it is sending me the error that the inputPlayer is null. (But it shouldn't)

**Comment by ZakShearman** [comment]
Please can you attach more code from your command, preferably full or at least including all parts where the argument is used.

Also the error and what line it references

Your code attached is confusing as you create inputPlayer but then reference inputGroup for the autocomplete?

**Reply by DevBlokJochem** [reply]
...

**Reply by DevBlokJochem** [reply]
I'm struggling for days, and the mistake could be that the tabcompletion is filled in from the wrong variable

**Comment by DevBlokJochem** [comment]
This is my WarstomCommand: https://paste.learnspigot.com/hoyoqoneka.kotlin
And this is my subcommand: https://paste.learnspigot.com/usulamoyit.kotlin
And this is the error on line 20: https://paste.learnspigot.com/odoxivefox.swift
